specs/001-develop-streaming-indicators/checklists/buffer-list.md

```diff
@@ -137,6 +137,29 @@
 - [ ] CHK095 - Are streaming usage example requirements specified for indicator docs? [Completeness, Plan §Documentation Updates]
 - [ ] CHK096 - Are requirements for documenting non-standard cache pruning behavior specified? [Completeness, Instructions §Auto-pruning]
 
+## Website Indicator Documentation Requirements
```

Comment:
Include checklist items for

- update migration docs
- update spec kit task progress